src/ui/component/userEmailNew/view.jsx

```diff
@@ -26,20 +25,15 @@ function UserEmailNew(props: Props) {
     setSync(formSyncEnabled);
     addUserEmail(newEmail);
     analytics.emailProvidedEvent();
-
-    // @if TARGET='web'
-    Lbryio.call('user_tag', 'edit', { add: 'lbrytv' });
-    // @endif
   }
 
   return (
     <React.Fragment>
-      <h1 className="section__title--large">{__('Welcome To LBRY')}</h1>
+      <h1 className="section__title--large">{__('Sign In To LBRY')}</h1>
```

Comment:
Sign in to LBRY or Sign In to LBRY

Typically in would not be capitalized but since it is the proper name of a page, it arguably could be and may look more natural